README.md

```diff
@@ -1 +1,120 @@
+- [Future Features ](#future-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+# Countries Weather <a name="about-project"></a>
+
+Countries Weather is a React project that provides users with up-to-date weather information for countries around the world. It offers a user-friendly interface where users can easily search for a specific country and view its current weather conditions, including temperature, humidity, wind speed, and more.
+
+
+
+### 🛠 Built With <a name="built-with"></a>
+- HTML
+- SCSS
+- React
+- Redux/toolkit
+- Weather API
+
+### Key Features <a name="key-features"></a>
+
+- [ ] **Show all counties and filter them.**
+- [ ] **Represent weather for each country.**
+
+### Future Features <a name="future-features"></a>
+
+- Geolocation: Implement geolocation functionality to automatically detect the user's location and display the local weather by default.
+- Favorites: Enable users to save their favorite countries and easily access their weather information without having to search for them repeatedly.
+- Weather Alerts: Integrate weather alert notifications to inform users about severe weather conditions in their selected countries.
+- Map Visualization: Incorporate a map interface to visualize the weather data geographically and enhance the user's understanding of global weather patterns.
+
+### 🚀 Live Demo <a name='live-demo'></a> 
+- [Experince a live demo](https://countries-weather-gtwj.vercel.app)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- a browser
+- git
+- nodejs
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/ybadr99/countries-weather.git
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following commands:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+  npx hint .
+  npx eslint .
+```
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Yousef Badr
+> 👤 **Author1**
+
+- GitHub: [@ybadr99](https://github.com/ybadr99)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/yousef-mohamed-badr/)
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Feel free to check the [issues page](https://github.com/ybadr99/math_magicians/issues).
+Contributions, issues, and feature requests are welcome!
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project:thumbsup:
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] As mentioned the previous reviewer, `Kindly an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).` This requirement is yet to be implemented. Kindly do just that and have your project approved.